query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java

```diff
@@ -291,6 +291,21 @@ public void testServicesQueries() {
     assertTrue(serviceNames.isEmpty());
   }
 
+  @Test
+  public void testServicesQueriesForAvgRate() {
+    LOG.info("Services queries for AVG_RATE");
+    Iterator<ResultSetChunk> itr = queryServiceClient.executeQuery(
+        ServicesQueries.buildAvgRateQuery(), TENANT_ID_MAP, 10000);
+    List<ResultSetChunk> list = Streams.stream(itr).collect(Collectors.toList());
+    List<Row> rows = list.get(0).getRowList();
+    assertEquals(4, rows.size());
```

Comment:
While reading the test `4` seems very arbitrary. Could we add a comment or make it explicit where this `4` comes from?

Reply:
We know this 4 from running the actual query on pinot console.